### DISCUSSION TITLE
How to use Symbolic Link

### DISCUSSION THREAD

**Discussion by 6643** [solidjs/solid#2305] [Q&A]
### Describe the bug

I have a folder that all projects need to use
Use `ln /_/lib/solid/widget/* . widget`
Can run normally
Use `ln -s /_/lib/solid/widget . widget`
Running error: Unable to parse 'solid js' in'/_/lib/solid/widget '
How to use Symbolic Link

### Your Example Website or App

nil

### Steps to Reproduce the Bug or Issue

nil

### Expected behavior

nil

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by ryansolid** [comment]
Maybe ask as a discussion? This isn't so much a bug but a request for advice based on a specific platform/platform feature. I'm not sure of the answer myself and the issue seems to be missing information. I have moved this to a discussion for you.

**Reply by trusktr** [reply]
Yeah, no idea what the issue is. Please post working example of the problem.
